presto-docs/src/main/sphinx/develop/connector.rst

```diff
@@ -0,0 +1,58 @@
+==========
+Connectors
```

Comment:
We should name the file `connectors` (plural) to be consistent with the other files